tests/src/unit/WebdamConfigFormTest.php

```diff
@@ -25,7 +26,6 @@ public function setUp() {
     \Drupal::setContainer($container);
   }
 
-
   public function testGetFormId() {
     $form = new WebdamConfig($this->getConfigFactoryStub());
```

Comment:
This is where your tests are failing. It's only passing a ConfigFactory stub into the constructor, but since you've changed the constructor arguments, it now needs a second argument.